### PR TITLE
Issues/1 - B2 and WS now builds via Gradle

### PR DESCRIPTION
**Pull request by cbeach47**
I don't have a v3200 / v3300 series Blackboard to test with, so the _structure_ of the building block and web services are now on par with the old ant builds.  Once this is merged, I'll branch the code and change the master branch to work with v3400 series Blackboard.

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+charset = utf-8
+end_of_line = lf
+insert_final_newline = true
+indent_style = space
+indent_size = 2
+trim_trailing_whitespace = true
+
+[*.java]
+indent_style = tab
+tab_width = 4
```
Comment (ChristianMurphy) [inline .editorconfig:13]: :1st_place_medal: Editor config is a good add for the project

File: oeqCommon/.classpath [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (ChristianMurphy) [inline oeqCommon/.classpath]: Is classpath intended to be committed?

File: oeqCommon/.project [hunk 3]
```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (ChristianMurphy) [inline oeqCommon/.project]: Is the project file intentionally included?

File: build.gradle [hunk 4]
```diff
@@ -0,0 +1,131 @@
+buildscript {
+  repositories {
+    jcenter()
+  }
+    
+  dependencies {
+    // Needed for pre-compiling the JSPs
+    classpath "com.bmuschko:gradle-tomcat-plugin:${gradleTomcatPluginVersion}"
```
Comment (ChristianMurphy) [inline build.gradle:8]: Neat :+1: 
this is something uPortal looked at briefly but haven't had a chance to adopt yet.

**Review by ChristianMurphy — APPROVED** [review]
`build.gradle` looks good

File: oeqCommon/.project (hunk 3 above)
Comment (cbeach47) [reply oeqCommon/.project]: ah no - thanks for the catch

File: oeqCommon/.classpath (hunk 2 above)
Comment (cbeach47) [reply oeqCommon/.classpath]: no - thanks for the catch

**Pull request merged**